go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/gophercloud/gophercloud v1.14.0
 	github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus v1.0.1
-	github.com/hashicorp/vault/api v1.14.0
+	github.com/hashicorp/vault/api v1.15.0
```

Comment:
I may be wrong, but does this have to be 1.15.5 according to the CVE patch fix?

Reply:
Oh wow, I misread it as 1.15.0. That is a really good catch, thanks!